TombEngine/Game/effects/tomb4fx.cpp

```diff
@@ -1603,7 +1603,7 @@ void UpdateShockwaves()
 								angle,
 								sw->speed);
 
-							DoDamage(LaraItem, sw->speed >> (((sw->flags >> 1) & 1) + 2));
+							DoDamage(LaraItem, sw->speed >> (((sw->damage >> 1) & 1) + 2));
```

Comment:
If flags doesnt exist anymore and you use `damage` field, you may directly apply damage without bitshift `>> 1`. Just don't forget to adjust damage (`>> 1` basically means "divide by 2", so you must divide provided damage by 2).

Reply:
now real damage values are used